packages/google_maps_flutter/google_maps_flutter_web/example/integration_test/google_maps_controller_test.dart

```diff
@@ -145,8 +145,8 @@ void main() {
           expect(() {
             controller.updateCircles(
               CircleUpdates.from(
-                <Circle>{},
-                <Circle>{},
+                const <Circle>{},
+                const <Circle>{},
```

Comment:
(A fix for this just got pushed to `main`, you should get it when you rebase next. Apologies for the breakage!)

Reply:
I think this is dealt with